File: containers/python/code.py
def main(arr , num) :
    # while True :
    #
    #     print('sup')
    #     pass
    for i in range(len(arr)) :
        for j in range(len(arr)) :
            if arr[i] + arr[j] == num :
                return [i , j]
    return None


import sys
import json

args = json.loads(sys.argv[1])


data = {
    'type' : 'result' ,
    'content' : main(*args),
}

print(json.dumps(data))

File: containers/typescript/code.ts
function isPalindrome(n:number):boolean{
  for(;;){
    console.log('.' )
  }   
  return n.toString().split('').reverse().join('') === n.toString() 
}

//the toppings ...
const args : any[]= JSON.parse(process.argv[2]);
console.log(args)
// add the type (func (...args as type)
//add when formatting the code in the main server 
type ArgType = [number] 
const data = {
    'type' : 'result' ,
    'content' :isPalindrome(...args as ArgType),
}

console.log()
console.log(JSON.stringify(data)); 
